todo_app/data/trello_items.py

```diff
@@ -5,76 +5,65 @@
 load_dotenv()
 trello_key=(os.getenv('TRELLO_KEY'))
 trello_token=(os.getenv('TRELLO_TOKEN'))
-#Get Trello items
-def get_items():
-   
-  url = "https://api.trello.com/1/boards/62f2b96de0b99c00aff27ae6/lists?"
+trello_todo_list=(os.getenv('TRELLO_TODO_LIST'))
+trello_done_list=(os.getenv('TRELLO_DONE_LIST'))
+trello_board_url=(os.getenv('TRELLO_BOARD_URL'))
 
+def get_items():   
+  url = trello_board_url
```

Comment:
Minor, but it might be easier to ask the user just to specify the board id itself rather than the full URL, e.g. something like:

```
    url = f"https://api.trello.com/1/boards/{os.getenv("TRELLO_BOARD_ID)}/lists"
```